lib/bone.js

```diff
@@ -928,6 +928,25 @@ class Bone {
       }
     }
 
+    const primaryKeyName = Object.keys(attributes).find(key => attributes[key].primaryKey);
+    if (primaryKeyName && primaryKeyName !== this.primaryKey) {
```

Comment:
s/primaryKeyName/primaryKey/

the convention is, primaryKey -> fooBar, primaryColumn -> foo_bar